### PR TITLE
Initial Y6 update

### PR DESCRIPTION
**Pull request by kai-herron**

### CODE REVIEW

File: lsbgalfit/cutout.py [hunk 1]
```diff
@@ -10,21 +10,169 @@
...
+#from astropy.stats import sigma_clipped_stats
 from astropy.nddata import Cutout2D
 warnings.simplefilter('ignore', UserWarning)
+#from astropy.utils.compat.optional_deps import HAS_BOTTLENECK
 
 import galsim
 from galsim.des.des_psfex import DES_PSFEx
 
 BANDS = ['g','r','i','z','Y','det']
 KEYWORDS = ['TILENAME','BAND','FILTER','MAGZERO','DESFNAME','UNITNAME','ATTNUM','REQNUM']
```
Comment (kadrlica) [inline lsbgalfit/cutout.py:30]: Do we want this to be in another utils file?

File: lsbgalfit/cutout.py [hunk 2]
```diff
@@ -95,35 +243,36 @@ def clear_segmap(segmap,obj,cat,bands,custom=None):
             segmap[idx] = 0
         except KeyError: pass
```
Comment (kadrlica) [inline lsbgalfit/cutout.py:245]: Ouch, this was a bad one!

**Review by kadrlica — APPROVED** [review]
Looks good to me! Maybe think about whether the sigma_clip_stats could belong in a utils file to be used elsewhere.

File: lsbgalfit/feedme.py [hunk 3]
```diff
@@ -86,7 +87,7 @@ def build_feedme(config, obj):
     #reffout = ','.join('%.2f'%obj[f'FLUX_RADIUS_{band.upper()}'] for band in bands)
     reff    = np.max([obj[f'FLUX_RADIUS_{band.upper()}'] for band in bands])
     reffstr = '%.2f'%reff
-    ellip   = obj['B_IMAGE']/obj['A_IMAGE']
+    ellip   = obj['B_IMAGE']/obj['A_IMAGE'] # should be called "elongation" or "ratio"
```
Comment (kadrlica) [inline lsbgalfit/feedme.py:90]: I agree, I think galfit talks about this as "axis ratio (b/a) in each band"

**Comment by kadrlica** [comment]
This looks good to me. Can you remind me why we need to monkey patch astropy's sigma clipping? Would it be sufficient to use a newer environment with a newer version of astropy?

**Comment by kai-herron** [comment]
I remember there being a divide by zero error and a newer version of sigma clipping had a way to handle it but I could be wrong... 
If we wanted to just run on an environment similar to des81, just adding a newer version of astropy that would remove the need for the monkey patch, I just worry about other dependencies being affected by updating the whole package

**Pull request merged**